react/react2/week1/todo-list-app/src/App.js

```diff
@@ -39,124 +43,28 @@ const App = () => {
       setIsEmpty(true)
     }
   }, [query, url])
+
+  const contextValue = {
+    inputHandler: inputHandler,
+    users: users,
+    title: 'GitHub user searcher',
+  }
+
   return (
-    <>
-      <Header title={'GitHub user searcher'} />
+    <UserContext.Provider value={contextValue}>
       <div className='container'>
-        <Input inputHandler={inputHandler} />
-        {isLoading && <ReaquestStatus status={'LOADING ...'} />}
-        {isError && <ReaquestStatus status={'ERROR ... 404. Bad request'} />}
+        <Header />
+        <Input />
+        {isLoading && <RequestStatus status={'LOADING ...'} />}
+        {isError && <RequestStatus status={'ERROR ... 404. Bad request'} />}
         {isEmpty ? (
```

Comment:
you can just check the length of the fetched array. You wouldn't need an additional "isEmpty" state